src/lib/cache/subCache.cpp

```diff
@@ -735,7 +737,21 @@ void subCacheItemInsert
 
   if (stringFilterP != NULL)
   {
-    cSubP->expression.stringFilter = *stringFilterP;  // Object copy
+    std::string  errorString;
+
+    if (!cSubP->expression.stringFilter.fill(stringFilterP, &errorString))
+    {
+      LM_E(("Runtime Error (%s)", errorString.c_str()));
+
+      //
+      // NOTE
+      //   Here, the cached subscription should have a String Filter but as clone failed, it doesn't
+      //   The subscription is already in mongo and hopefully this erroneous situation is fixed 
+      //   once the sub-cache is refreshed.
+      //
+      //   This 'but' should be minimized once the issue 2082 gets implemented.
+      //   [ Only reason for clone to fail (apart from out-of-memory) seems to be an invalid regex ]
```

Comment:
Appart of the LM_E (which I suggest to remove above) I understand that this `{ }` block is left empty just to put the comment (ok with that, I'm only checking if I'm understand correctly).